static/main.css

```diff
@@ -232,4 +232,70 @@ main.container {
 .create-event-button:hover {
   background-color: #D85536;
   border: solid 2px #D85536;
+}
+
+.white-background-button {
+  width: auto;
+  height: auto;
+  padding: 9px 20px 7px 19px;
+  border-radius: 24px;
+  border: solid 2px #e84a27;
+  background-color: #fff;
+  color:#002855;
+  font-weight: bold;
+  font-size: 14px;
+}
+
+.white-background-button:hover {
+  background-color: #fff;
+  border: solid 2px #e84a27;
+  color:#002855;
+}
+
+.orange-background-button {
+  width: auto;
+  height: auto;
+  padding: 9px 25px 7px 24px;
+  border-radius: 24px;
+  border: solid 2px #e84a27;
+  background-color: #e84a27;
+  font-weight: bold;
+  font-size: 14px;
+}
+
+.orange-background-button:hover {
```

Comment:
Same for orange-background-button:hover and orange-background-button and notification-button and grey-button

Reply:
This is intended to override the Bootstrap hover color as well.

Reply:
see if you can combine the css styles.

Reply:
I think we might need <button-class-name>:active css styles too. I think this would set the style when on-click (like click and hold). 
I'm not too sure on this. So might need to take @bingzhang 's opinion as well here.

Reply:
I have add the :active selector to CSS after discussing with @bingzhang and @sandeep-ps